pkgs/os-specific/linux/pam_dp9ik/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, tlsclient
+, stdenv
+, pkg-config
+, pam
+}:
+
+stdenv.mkDerivation rec {
+  inherit (tlsclient) src;
+
+  pname = "pam_dp9ik";
+  version = "unstable-2022-12-20";
```

```suggestion
  pname = "pam_dp9ik";
  inherit (tlsclient) src version;
```
since it is the same source, it is also the same version